jitsi-utils/pom.xml

```diff
@@ -127,32 +120,90 @@
+                                <source>src/main/java</source>
+                            </sourceDirs>
+                        </configuration>
                     </execution>
                 </executions>
                 <configuration>
                     <jvmTarget>1.8</jvmTarget>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>4.2.1</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <_removeheaders>Bnd-*, Tool, Require-Capability, Include-Resource
+                        </_removeheaders>
+                        <!--
+                        Prevent importing a specific version. Some libraries
+                        like Guava rapidly change major versions, which is a pain for
+                        downstream consumers wanting a different/newer version than
+                        used here.
+                        -->
+                        <Import-Package>*;version=!</Import-Package>
+                    </instructions>
+                </configuration>
+            </plugin>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.5.1</version>
                 <executions>
+                    <!-- Replacing default-compile as it is treated specially by maven -->
                     <execution>
-                        <id>compile</id>
+                        <id>default-compile</id>
+                        <phase>none</phase>
+                    </execution>
+                    <!-- Replacing default-testCompile as it is treated specially by maven -->
+                    <execution>
+                        <id>default-testCompile</id>
+                        <phase>none</phase>
+                    </execution>
+                    <execution>
+                        <id>java-compile</id>
                         <phase>compile</phase>
                         <goals>
                             <goal>compile</goal>
                         </goals>
                     </execution>
                     <execution>
-                        <id>testCompile</id>
+                        <id>java-test-compile</id>
                         <phase>test-compile</phase>
                         <goals>
                             <goal>testCompile</goal>
                         </goals>
                     </execution>
                 </executions>
             </plugin>
+            <!-- This plugin is necessary for free/extensions functions defined in the kotlin code to be
+            visible to kotlin code using this lib.  See https://youtrack.jetbrains.com/issue/KT-9164. -->
```

Comment:
I assume you tried the other method mentioned in this issue?
```
<Include-Resource>
     /=src/main/resources/,
     META-INF=target/classes/META-INF
 </Include-Resource>
```

Reply:
(That would go into <instructions> below line 141)

Reply:
Cool, looks like this worked, I'll do it this way.